templates/base.html

```diff
@@ -224,8 +223,6 @@
     </section>
 
     {% block js %}
-        <script src="https://cdn.ravenjs.com/3.17.0/raven.min.js" crossorigin="anonymous"></script>
```

Comment:
I think this removes our Sentry-tracking from all frontend-code? We should perhaps migrate to the [Javascript version](https://docs.sentry.io/platforms/javascript/) instead?

Reply:
Yes. You are right. I didn't think about the JS part of the application. Will fix.